slack/web/base_client.py

```diff
@@ -475,7 +475,7 @@ def _perform_urllib_http_request(
                     resp = urlopen(  # skipcq: BAN-B310
                         req, context=self.ssl, timeout=self.timeout
                     )
-                charset = resp.headers.get_content_charset()
+                charset = resp.headers.get_content_charset() or "utf-8"
```

Comment:
Why not add the type annotation?

```
charset: str = resp.headers.get_content_charset() or "utf-8"
```

Reply:
Good point. I'll update it later for consistency.